src/code42cli/file_readers.py

```diff
@@ -20,19 +20,28 @@ def read_csv(file, headers=None):
     """Helper to read a csv file object into dict rows, automatically removing header row
     if it exists, and errors if column count doesn't match header list length.
     """
-    reader = csv.DictReader(file, fieldnames=headers)
+    reader = csv.DictReader(file)
     first_row = next(reader)
-    if None in first_row or None in first_row.values():
```

Comment:
Would like to understand what has changed here, does it addresses a special case? if so, a unit test would suffice to explain the same.

Reply:
Previously, the DictReader required that the CSV have a precise number of columns, and that the columns be in the correct order, but it did not require column headers. This change allows for files formatted in that way, but also allows for CSVs with an arbitrary number and order of columns as long as the correct headers are included.

I'll add some tests to clarify.

Reply:
I've added test_file_readers.py and two tests to cover the CSV cases this change supports.